tls/extensions/s2n_extension_type_lists.c

```diff
@@ -109,50 +97,52 @@ static const s2n_extension_type *const tls12_server_hello_extensions[] = {
  *# -  key_share (see Section 4.2.8)
  */
 static const s2n_extension_type *const hello_retry_request_extensions[] = {
-        &s2n_server_supported_versions_extension,
-        &s2n_server_cookie_extension,
-        &s2n_server_key_share_extension,
+    &s2n_server_supported_versions_extension,
+    &s2n_server_cookie_extension,
+    &s2n_server_key_share_extension,
 };
 
 static const s2n_extension_type *const tls13_server_hello_extensions[] = {
-        &s2n_server_supported_versions_extension,
-        &s2n_server_key_share_extension,
-        &s2n_server_psk_extension, /* MUST appear after keyshare extension */
+    &s2n_server_supported_versions_extension, &s2n_server_key_share_extension,
+    &s2n_server_psk_extension, /* MUST appear after keyshare extension */
 };
 
 static const s2n_extension_type *const encrypted_extensions[] = {
-        &s2n_server_server_name_extension,
-        &s2n_server_max_fragment_length_extension,
-        &s2n_server_alpn_extension,
-        &s2n_quic_transport_parameters_extension,
-        &s2n_server_early_data_indication_extension,
+    &s2n_server_server_name_extension,
+    &s2n_server_max_fragment_length_extension,
+    &s2n_server_alpn_extension,
+    &s2n_quic_transport_parameters_extension,
+    &s2n_server_early_data_indication_extension,
 };
 
 static const s2n_extension_type *const cert_req_extensions[] = {
-        &s2n_server_signature_algorithms_extension,
+    &s2n_server_signature_algorithms_extension,
 };
 
 static const s2n_extension_type *const certificate_extensions[] = {
-        &s2n_tls13_server_status_request_extension,
-        &s2n_server_sct_list_extension,
+    &s2n_tls13_server_status_request_extension,
+    &s2n_server_sct_list_extension,
 };
 
 static const s2n_extension_type *const nst_extensions[] = {
-        &s2n_nst_early_data_indication_extension,
+    &s2n_nst_early_data_indication_extension,
 };
 
-#define S2N_EXTENSION_LIST(list) { .extension_types = (list), .count = s2n_array_len(list) }
+#define S2N_EXTENSION_LIST(list)                                \
+    {                                                           \
+        .extension_types = (list), .count = s2n_array_len(list) \
+    }
```

Comment:
This isn't ideal, but I doubt we can change it

Reply:
Yeah this changes even with col-limit zero